scripts/kb_scripts/gwf_to_scs.py

```diff
@@ -66,10 +66,15 @@ def check_status(self):
 
 if __name__ == "__main__":
   parser = argparse.ArgumentParser(description='Converts directory with GWF files into SCs files')
-  parser.add_argument('input', action='store',
+  parser.add_argument('kb_path', action='store',
                       help='Path to input directory, that contains gwf files')
+  parser.add_argument('repo_path', action='store',
+                      help='Repo path file')
   args = parser.parse_args()
-  args.output =os.path.join(args.input, 'converted_gwf_to_scs')
+  args.output =os.path.join(args.kb_path, 'converted_gwf_to_scs')
```

Comment:
.scs files from .gwf should be placed near sources .scs file, not in separate folder